BookPlayer/Player/Views/ArtworkControl.swift

```diff
@@ -54,10 +60,23 @@ class ArtworkControl: UIView, UIGestureRecognizerDelegate {
         get {
             return self.artworkImage.image
         }
+
         set {
             self.artworkImage.image = newValue
 
-            self.artworkImage.resizeToFit()
+            let ratio = self.artworkImage.imageRatio
+            let base = min(self.artworkContainer.bounds.width, self.artworkContainer.bounds.height)
+
+            if ratio > 1 {
```

Comment:
Same comment as before, I think we can replace this `if ... else if ... else` for these two lines:
```
self.artworkWidth.constant = ratio > 1 ? base / ratio ? base
self.artworkHeight.constant = ratio < 1 ? base * ratio ? base
```